server/server.js

```diff
@@ -163,21 +163,65 @@ app.post(`${apiBaseUrl}/employees`,
         this.push(null);
       },
     });
+
     const columnMap = {
-      maximusId: 'maximusId',
-      eligibility: 'eligibility',
-      firstName: 'firstName',
-      lastName: 'lastName',
-      phoneNumber: 'phoneNumber',
-      emailAddress: 'emailAddress',
-      postalCode: 'postalCode',
-      preferredLocation: 'preferredLocation',
-      consent: 'consent',
+      ClientID: 'maximusId',
+      Lastname: 'lastName',
+      FirstName: 'firstName',
+      AddressLine1: 'addressLine1',
+      AddressLine2: 'addressLine2',
+      City: 'city',
+      Province: 'province',
+      Postal: 'postalCode',
+      Phone1: 'phoneNumber',
+      Email: 'emailAddress',
+      registered: 'registered',
+      CreatedDatetime: 'createdDatetime',
```

Comment:
I think we might want to name this maximusCreatedDatetime

Just to distinguish it from our own timestamp when we perform the import and it keep it really clear what this is, vs. the "registered" field which is capturing the timestamp of when the original form submission was made

Heck, we could rename both fields. originalSubmissionDatetime and maximusReceivedDatetime or something?